packages/flutter/test/cupertino/segmented_control_test.dart

```diff
@@ -277,8 +283,9 @@ void main() {
       ),
     );
 
-    final dynamic childList =
-        getRenderSegmentedControl(tester).getChildrenAsList();
+    await tester.pumpAndSettle();
+
+    final dynamic childList = getRenderSegmentedControl(tester).getChildrenAsList();
```

Comment:
generally we want to avoid using `dynamic` as much as possible, fwiw. We get no static type checking when we use `dynamic`.